1.1/translations/validator_fr.ttl

```diff
@@ -0,0 +1,866 @@
+		@prefix geo: <http://www.opengis.net/ont/geosparql#> .
+		@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+
+		<geometryCollection-x> 
+		    a geo:GeometryCollection ;
+		    rdfs:member <geometry-x> ;
+		.		
+		
+		<feature-x> geo:hasGeometry <geometry-x> .
+		"""@fr ,
+		"""
+		# Un exemple non valide: le noeud d'instance geo:GeometryCollection comporte un rdfs:membre sortant qui pointe a un noeud geo:Geometry and un autre noeud qui n'est pas un geo:Geometry 
+		@prefix geo: <http://www.opengis.net/ont/geosparql#> .
+		@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+
+		<geometryCollection-x> 
+		    a geo:GeometryCollection ;
+		    rdfs:member <geometry-x> , <feature-x> ;
+		.
+		
+		<feature-x> geo:hasGeometry <geometry-x> .
+		"""@fr ;
+.
+
+#################################################
+# Shape 23 - geo:SpatialObjectCollection
+#################################################
+
+<S23-SpatialObjectCollection-member-spatialObject>
+	a sh:NodeShape ;
+	sh:property <S23-SpatialObjectCollection-minOneMember-spatialObject-sub> , <S23-SpatialObjectCollection-member-onlySpatialObject-sub> ;
+	sh:targetClass geo:SpatialObjectCollection ;
+.
+
+<S23-SpatialObjectCollection-minOneMember-spatialObject-sub>
+	a sh:PropertyShape ;
+	sh:path rdfs:member ;
+	sh:minCount 1 ;
+	sh:message "Une instance de geo:eo:SpatialObjectCollection doit avoir au moins une relation sortante rdfs:member."@fr ;
+	skos:example 
+		"""
+		# Un exemple valide: Le noeud geo:SpatialObjectCollection a une relation sortante rdfs:member.
+		@prefix geo: <http://www.opengis.net/ont/geosparql#> .
+		@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+
+		<spatialObjectCollection-x>
+		    a geo:SpatialObjectCollection ;
+		    rdfs:member <feature-x>
+		.
+		
+		<feature-x> geo:hasGeometry <geometry-x> .
+		"""@fr ,
+		"""
+		# Un exemple valide: Le noeud geo:SpatialObjectCollection n'a pas une relation sortante rdfs:member.
+		@prefix geo: <http://www.opengis.net/ont/geosparql#> .
+		@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+
+		<spatialObjectCollection-x>
+		    a geo:SpatialObjectCollection ;
+		.
+		"""@fr ;
+.
+
+<S23-SpatialObjectCollection-member-onlySpatialObject-sub>
+	a sh:PropertyShape ;
+	sh:path rdfs:member ;
+	sh:sparql [
+		sh:message "Une instance de geo:SpatialObjectCollection ne doit avoir que des relations sortante rdfs:member qui pointes à des instances geo:SpatialObject ou ses sous-classes."@fr ;
+		sh:select """
+			PREFIX geo: <http://www.opengis.net/ont/geosparql#>
+			PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+			SELECT $this
+			WHERE {
+				$this rdfs:member ?spatialObject .
+				FILTER NOT EXISTS {
+					?spatialObject geo:hasGeometry|^geo:hasGeometry ?geometryOrFeature .
+				}
+			}
+		""" ;
+	] ;
+	skos:example 
+		"""
+		# Exemple valide: Le noeud geo:SpatialObjectCollection a des relations sortante rdfs:member qui pointes à des instances geo:SpatialObject
```

Comment:
"qui pointent" instead of "qui pointes" (plural verb)

Reply:
Pesky grammar!